packages/python/plotly/plotly/tests/test_core/test_subplots/test_make_subplots.py

```diff
@@ -1542,6 +1544,35 @@ def test_subplot_titles_insets(self):
         )
         self.assertEqual(fig, expected)
 
+    def test_subplot_titles_array(self):
```

Comment:
All current tests are using tuples as subplot_titles argument. I thought it would be a good idea to add a test which checks if it works the same with python array.